apps/explorer/src/components/transactions/index.tsx

```diff
@@ -79,6 +79,7 @@ export function Transactions({
         <div>
             {recentTx ? (
                 <TableCard
+                    fill={false}
```

Comment:
Why isn't this full width

Reply:
If we leave this fullWidth, it will stretch the table out and each column can be alot larger than what they need to be. For the main index table, if we want the columns to hug the content, we should let the column/cell determining the size of the table. 

First pic is from `production` and second is from this PR

<img width="745" alt="Screenshot 2023-04-12 at 4 27 27 PM" src="https://user-images.githubusercontent.com/127577476/231607613-6e527521-fa52-4862-99ec-e66073dec877.png">


<img width="685" alt="Screenshot 2023-04-12 at 4 27 30 PM" src="https://user-images.githubusercontent.com/127577476/231607596-7123477c-a4be-4d5c-b617-6f12d164dbf3.png">

Reply:
I imagine this should be consistent across the app though, why is it is special cased on the homepage?

Reply:
For the Validators table, if we do not stretch it, then it will end up with a lot of white space on the right. 

<img width="793" alt="Screenshot 2023-04-12 at 5 01 00 PM" src="https://user-images.githubusercontent.com/127577476/231611657-67f17a54-645c-4d06-897c-c1c9ba26054b.png">